etc/kayobe/pulp.yml

```diff
@@ -32,8 +32,14 @@ pulp_proxy_url: "{{ omit }}"
 ###############################################################################
 # StackHPC Pulp server
 
+# The scheme - http/https of the StackHPC Pulp service.
+stackhpc_release_pulp_scheme: "https://"
```

Comment:
nit: I think strictly the scheme is https

Reply:
Fixed